tracegen/internal/tracegen/config.go

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2018 The Jaeger Authors.
+// Copyright The OpenTelemetry Authors
```

Comment:
Please place OpenTelemetry copyright notice at the top and follow it by original Jaeger copyright notice (without changing anything in Jaeger notice).

Reply:
Done